iaso/api/payments/views.py

```diff
@@ -373,13 +381,16 @@ class PotentialPaymentsViewSet(ModelViewSet, AuditMixin):
     http_method_names = ["get", "head", "options", "trace"]
 
     def get_queryset(self):
-        return (
+        queryset = (
             PotentialPayment.objects.prefetch_related("change_requests")
             .filter(change_requests__created_by__iaso_profile__account=self.request.user.iaso_profile.account)
             # Filter out potential payments already linked to a payment lot as this means there's already a task running converting them into Payment
             .filter(payment_lot__isnull=True)
             .distinct()
         )
+        queryset = queryset.annotate(change_requests_count=Count("change_requests"))
```

Comment:
It seems to me that the missing annotation should've been detected in unit tests.